src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.ts

```diff
@@ -9,4 +9,5 @@ import { SidebarCardElement, SidebarTypes } from 'app/types/sidebar';
 export class SidebarCardItemComponent {
     @Input() sidebarItem: SidebarCardElement;
     @Input() sidebarType?: SidebarTypes;
+    @Input() groupKey?: string;
```

Comment:
New client guidelines want us to use of input <String>()
```suggestion
    groupKey = input<string>()
```